LayoutTests/http/tests/navigation-api/different-origin-entries-removed-after-bfcache-expected.txt

```diff
@@ -0,0 +1,7 @@
+PASS successfullyParsed is true
+
+TEST COMPLETE
```

Comment:
If `TEST COMPLETE` shows in the middle of the test and not at the end then the test is not correctly written.